cmd/plugin/cli/uninstall.go

```diff
@@ -140,6 +142,12 @@ func setUninstallValues(cmd *cobra.Command, config *apiv1.KubectlStorageOSConfig
 		if err != nil {
 			return err
 		}
+		config.Spec.IncludeLocalPathProvisioner, err = cmd.Flags().GetBool(installer.IncludeLocalPathProvisionerFlag)
+		if err != nil {
+			return err
+		}
+		fmt.Printf("include local path provisioner flag val %v\n", config.Spec.IncludeLocalPathProvisioner)
```

Comment:
I'm guessing this was a debug comment?

Reply:
Ah yes, there's some clean up that I need to do